.gitignore

```diff
@@ -4,6 +4,7 @@ bin/wasm2asm
 bin/s2wasm
 bin/wasm-as
 bin/wasm-dis
+lib/libbinaryen-c.so
```

Comment:
Hmm, probably we should change that too, I suppose. Anyhow, doesn't need to be in this PR.